acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/clique/CliqueMiningAcceptanceTest.java

```diff
@@ -42,6 +42,14 @@ public void shouldMineTransactionsOnSingleNode() throws IOException {
     cluster.verify(receiver.balanceEquals(3));
   }
 
+  @Test
+  public void shouldNotMineBlocksIfNoTransactionsOnSingleNode() throws IOException {
+    final BesuNode minerNode = besu.createCliqueNoEmptyBlockNode("miner1");
+    cluster.start(minerNode);
+
+    cluster.verify(clique.noNewBlockCreated(minerNode));
```

Comment:
Would also be good to have a test showing that when we have transactions again that we start mining blocks again

Reply:
godd point, will add this test too

Reply:
@jframe test added, please re-review